site/guide/work-with-model-findings.qmd

```diff
@@ -60,29 +60,65 @@ Filters can be removed by clicking on the {{< fa xmark >}} next to them on the m
 
 ## Add project findings
 
-As part of validation process, you may find issues with the model documentation that must be resolved. To indicate that there is an issue and to track the resolution, you add a new finding:
+As part of the validation process, you may find issues with the model documentation that must be resolved. To indicate that there is an issue and to track the resolution, you add a new finding. 
+
+- Findings are logged in your **Model Inventory**, within a model's documentation. 
+- Within the {{< var vm_platform >}}, you're able to add findings both on the overarching documentation overview page, and within each documentation section itself. 
 
 1. [Log in to the ValidMind UI]({{< var vm_url >}}).
 
 2. In the left sidebar, click [**Model Inventory**]({{< var vm_url >}}/model-inventory).
 
-3. Select a model by clicking on it or find your model by applying a filter or searching for it.
+3. Select a model by clicking on it or find your model by [applying a filter](#filter-project-findings) or searching for it.
+
+4. In the left sidebar that appears for your model, click **Documentation**. 
+
+::: {.callout-tip appearance="minimal"}
+You can now either log a finding on this overview page, or via a specific documentation section. Both methods will allow you to associate a finding with a documentation section if desired. 
+:::
 
-4. In the left sidebar that appears for your model, click **Documentation**.
+#### Add finding via overview
 
-5. On the page that opens, click **Add Finding**.
+To log a finding from the documentation overview: 
 
-6. On the **Add Project Finding** page that opens,  provide information for: 
+1. On your model's **Documentation** page, click **Add Finding**.
+
+2. On the **Add Model Finding** page that opens, provide information for: 
 
    - Title
    - Risk area
    - Owner
    - Severity
    - Due date
+   - Documentation section
    - Description
 
    At a minimum, you must provide the required information indicated by `*`.
 
+3. When you are done, click **Save** to submit the finding. 
+
+#### Add finding via section
+
+While working within a section of your documentation, you can easily log a finding associated with that section without leaving the page:
+
+1. Click on **ValidMind Insights™** to expand the insight panel.
+
+2. For the section you want to add a Finding for, click **{{< fa triangle-exclamation >}}  Add Finding** beneath the Documentation Guidelines.
+
+3. On the **Add Model Finding** page that opens,  provide information for: 
+
+   - Title
+   - Risk area
+   - Owner
+   - Severity
+   - Due date
+   - Documentation section
+   - Description
+
+   At a minimum, you must provide the required information indicated by `*`. **Documentation section will auto-populated with the section you are working from** — you are able to select another section if desired.
```

Comment:
So, technically, `*` should be bolded as it's a UI element, but good luck bolding an asterisk unless you drop in HTML. (I'd maybe consider skipping that as you're explaining how the UI works? Using asterisks to indicate required information shouldn't really need explaining at this point, unless you've not used a computer since they got rid of the red boxes that indicate death notices. 🙂) Also, the follow-on sentence had a grammatical snafu of some sort, but check if the fix works for you. 

```suggestion
   At a minimum, you must provide the required information indicated  indicated by <b>*</b>. **The documentation section will be auto-populated with the section you are working from** — you are able to select another section if desired.
```